opal/mca/common/ucx/common_ucx.c

```diff
@@ -126,6 +127,59 @@ OPAL_DECLSPEC void opal_common_ucx_mca_deregister(void)
     opal_output_close(opal_common_ucx.output);
 }
 
+OPAL_DECLSPEC int opal_common_ucx_priority(ucp_context_h context, int min, int max)
+{
+#if HAVE_DECL_OPEN_MEMSTREAM
+    static const char *transports[] = {
```

Comment:
Should we make the list of UCX transports an mca parameter so that users could use tcp in UCX if they want?

Reply:
yes, as Jeff commented, will add mca param